lib/VeresOne.js

```diff
@@ -297,26 +291,22 @@ class VeresOne {
   /**
    * Adds an ocap invocation proof to an operation.
    *
-   * TODO: support `passphrase` for encrypted private key pem or keep decrypt
-   * as the responsibility of the caller?
-   *
-   * FIXME: use ldocap.js
+   * TODO: use ocapld.js
```

Comment:
This TODO can be removed as well?  Looks like ocapld is being used.  Unless that means something else?

Reply:
Oh, whoops! I didn't even see that it WAS being used! Removed.